pyfolio/perf_attrib.py

```diff
@@ -203,11 +203,7 @@ def perf_attrib(returns,
         factor_returns = factor_returns.drop(missing_factor_loadings_index,
                                              errors='ignore')
 
-    if pos_in_dollars:
-        # convert holdings to percentages
-        positions = get_percent_alloc(positions)
-
-    if transactions is not None:
+    if transactions is not None and pos_in_dollars:
```

Comment:
Is it intentional that this disables the warning if `pos_in_dollars` is False?

Reply:
@abhijeetkalyan @yankees714 ^

Reply:
Yep, this is intentional - currently `get_turnover()` doesn't support positions as percentages, but it probably wouldn't be too hard to add support for that.